src/govuk/common/get-dataset.mjs

```diff
@@ -0,0 +1,14 @@
+import '../vendor/polyfills/Element/prototype/dataset.mjs'
+
+/**
+ * Access the elements dataset
+ *
+ * This is a functional wrapper around `element.dataset` to centralise the
+ * import of the dataset polyfill
+ *
+ * @param {HTMLElement|SVGElement} element - The element whose dataset to access
```

Comment:
Nice to add **SVGElement** here but I have a feeling we might see conflicting types downstream, especially with `document.activeElement` returning the base `Element`

Can I suggest we leave out `getDataset()` from this PR and stick to the reorganising for now?